app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,22 @@
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">The Clock</a>
+          <ul class="navbar-nav d-flex">
+<% if user_signed_in? %>
+            <li class="nav-item">
+  <%= link_to 'Sign out', destroy_user_session_path, method: :delete, class: "nav-link" %>
+            </li>
+            <li class="nav-item">
+  <%= link_to 'Edit my account', edit_user_registration_path, class: "nav-link" %>
+            </li>
+<% else %>
+            <li class="nav-item">
+              <%= link_to 'Sign in', new_user_session_path, class: "nav-link" %>
+            </li>
+            <li class="nav-item">
+              <%= link_to 'Register', new_user_registration_path, class: "nav-link"%>
+            </li>
+<% end %>
```

Comment:
review the indentation here please

Reply:
Thanks a lot for all the comments, I really forgot to solve that problems with indentation that I had.